feature-libs/checkout/base/assets/translations/en/checkout.i18n.ts

```diff
@@ -10,13 +10,13 @@ export const checkout = {
   },
   checkoutProgress: {
     label: 'Checkout Progress',
-    deliveryAddress: 'Delivery Address',
```

Comment:
Changing labels are breaking changes.  Need entries in migration doc.

Reply:
Added